config/routes.rb

```diff
@@ -274,4 +288,5 @@
   match ':controller(/:action(/:id))(.:format)'
```

Comment:
ended up leaving the generic match rule in because over 20 test cases depended on this to pass.  As far as I can tell, it doesn't affect the functionality of the RESTful api, since the route is set up before all the others (and therefore is a priority match over the general rule)